bigbluebutton-html5/imports/startup/client/logger.js

```diff
@@ -66,10 +66,15 @@ function createStreamForTarget(target, options) {
 }
 
 function generateLoggerStreams(config) {
-  return config.map(({ target, level, ...streamOptions }) => ({
-    level,
-    stream: createStreamForTarget(target, streamOptions),
-  }));
+  let result = [];
+  Object.keys(config).forEach((key) => {
```

Comment:
Why not keep the same mapper and just filter out disabled ones?

```
  return config
    .filter(({ enabled }) => enabled)
	.map(({ target, level, enabled, ...streamOptions }) => ({
      level,
	  stream: createStreamForTarget(target, streamOptions),
	}));
```

Reply:
target is now the key of the {...}, not an inside property. I was not able to get the key without this workaround. Do you still find it possible?